<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bilan</class>
 <widget class="QDialog" name="bilan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>632</width>
    <height>540</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>632</width>
    <height>540</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Bilan</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../ressources/logo.png</normaloff>../ressources/logo.png</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="titre">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>161</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Bilan</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="tablePassif">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>140</y>
     <width>271</width>
     <height>311</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>650</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Compte</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Montant</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="totalCreanceLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>99</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Creances : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="totalCreanceLineEdit">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>460</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="totalDetteLabel">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>460</y>
     <width>90</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Dettes :   </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="totalDetteLineEdit">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>460</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelDette">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>110</y>
     <width>53</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <italic>false</italic>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Dettes: </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelCreance">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>70</width>
     <height>29</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Creances :</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>291</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="1" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="bilenPushButton">
        <property name="text">
         <string>Afficher Bilan</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="quitterPushButton">
        <property name="text">
         <string>Quitter</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labelMonnaie1">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>460</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>€</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelMonnaie2">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>460</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>€</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableActif">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>261</width>
     <height>311</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>650</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Compte</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Montant</string>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </column>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>500</y>
     <width>571</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="pushButtonExport">
      <property name="text">
       <string>Exporter en PDF</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quitterPushButton</sender>
   <signal>clicked()</signal>
   <receiver>bilan</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>277</x>
     <y>443</y>
    </hint>
    <hint type="destinationlabel">
     <x>354</x>
     <y>451</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
